### ISSUE TITLE
Change name of package

### ISSUE THREAD

**Issue by AleshaOleg** [csstools/postcss-sass#3]
Hey, @jonathantneal! I found your package while searching for my own:D I have a package with same name [postcss-sass](https://github.com/AleshaOleg/postcss-sass). I know you using a prefix `@csstools/` in order to distinguish our packages. But may be better to change name of your package, to not to mislead users? I speaked with @ai, he propose a name `postcss-with-sass`. What you think about?

**Comment by jonathantneal** [comment]
Hey @AleshaOleg, @ai,

I’m not closed to changing the name. Presently, I do think differently than the suggestion for the new name. Could we have a discussion about this?

**1. How should we name different kinds of postcss tools?**

For instance, we have **postcss-safe-parser** (suggesting a **parser** suffix), but then **postcss-scss** and **postcss-sass** do not follow this convention. Meanwhile, **postcss-selector-parser** is a different kind of parser altogether that isn’t really part of postcss. Our guidelines currently say plugin names should be straightforward — *If you wrote a plugin to support mixins, postcss-mixins would be a good name*. Do both postcss-sass projects follow this guideline? If so, let’s resolve that.

**2. Do @ namespaces make this a non-issue?**

I thought the point of namespaces was to protect us from worrying about naming conflicts. I chose a namespace because I want developers to expect the same level of consistency in the code, naming, testing, and development style in each of my projects.

**3. Should postcss adopt @ namespaces?**

I want to piggy-back on both of my questions. I think we should consider using the **postcss** namespace for postcss and whitelisted packages. Babel has moved this direction after experiencing [similar issues](https://babeljs.io/blog/2017/12/27/nearing-the-7.0-release#renames-scoped-packages-babelx). This will also make it easier to enforce consistent project naming. What do you think?

**Comment by ai** [comment]
> we have postcss-safe-parser (suggesting a parser suffix), but then postcss-scss and postcss-sass do not follow this convention

Because `postcss-scss` and `postcss-sass` are the syntaxes (parser + stringifier) and `postcss-safe-parser` doesn’t have stringifier.

> Do @ namespaces make this a non-issue?

I think users don’t remember that `@`-prefix. Anyway they think about project as `postcss-sass`.

For example, somebody tell you “to support Sass in Stylint you need to set `postcss-sass` syntax”, you google it and use first link and Stylint doesn‘t work because it was not a syntax :).

> I think we should consider using the postcss namespace for postcss and whitelisted packages.

It will work if we will have some quality standards. So `@postcss` prefix will mean somthing. Right now both `postcss-sass` and `@jonathantneal/postcss-sass` is unofficial. `@postcss` prefix could be useful, but not for this case.

**Comment by gavinmcfarland** [comment]
One of the issues we will encounter with PostCss namespacing is that because they can and I guess ought to be projects that do one thing well a lot of peoples projects will overlap. Naming a project with a straight forward name yet being unique will be a difficult task for some people. I have my own projects which I face this problem with.

**Comment by jonathantneal** [comment]
Closing due to inactivity and no continued concern from the community.

**Comment by gavinmcfarland** [comment]
I just thought I'd share some learnings with my own projects. I have decided it helps if the name is more descriptive. For example I have a plugin which allows you to create borders using box-shadows. I chose the name `postcss-border-align` because this is at the heart of what the plugin helps with while still being flexible for the future possibilities of my plugin. I could have called it `postcss-border` but it could have conflicted with someone else's plugin in the future and it's a bit ambiguous and doesn't tell you much about the plugin. Equally I could have called it `postcss-border-align-property` but this would have been quite lengthy and isn't completely accurate. Not all plugins will be this straight forward to name and unfortunately the fact is that if two plugins do something very similar of not the same thing, will ultimately lead to the likelihood of having the same name.
